src/WebUI/RunDetails.scss

```diff
@@ -1,3 +1,7 @@
 .runs-bullet {
     margin: 0 6px;
+}
+
+.run-name-link.bolt-table-link{
```

Comment:
low prio: is it possible to avoid usage of bolt-table-link? if so would you able to do so?